app/commands/user/profile/create.rb

```diff
@@ -0,0 +1,13 @@
+class User::Profile
+  class Create
+    include Mandate
+
+    initialize_with :user
+
+    def call
+      raise ProfileCriteriaNotFulfilledError unless user.profile_unlocked?
+
+      User::Profile.new(user:)
```

Comment:
This should actually create (as per app/controllers/api/profiles_controller.rb:L5).